pkg/apis/operators/shared/phase_types.go

```diff
@@ -0,0 +1,60 @@
+package shared
```

Comment:
I am assuming you did a `git mv` and git decided the file was new. Was that the case?

Reply:
That is correct.

Reply:
It decided to create / delete the phase_type after adding the generated deepcopy functions.